docker-compose-microservices.yml

```diff
@@ -0,0 +1,107 @@
+version: '2'
+services:
+    artemis-app:
+        image: ghcr.io/ls1intum/artemis-service
+        build:
+            context: .
+            dockerfile: ./src/main/docker/Dockerfile
+        volumes:
+            - ./src/main/resources/config/application-dev.yml:/home/jhipster/application-dev.yml:ro
+            - ./src/main/resources/config/application-artemis.yml:/home/jhipster/application-artemis.yml:ro
+        environment:
+            - _JAVA_OPTIONS=-Xmx512m -Xms256m
+            - SPRING_PROFILES_ACTIVE=dev,artemis,openapi
+            - MANAGEMENT_METRICS_EXPORT_PROMETHEUS_ENABLED=true
+            - SPRING_DATASOURCE_URL=jdbc:mysql://artemis-mysql:3306/Artemis?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC
+            - EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=http://admin:$${jhipster.registry.password}@jhipster-registry:8761/eureka
+            - SPRING_CLOUD_CONFIG_URI=http://admin:$${jhipster.registry.password}@jhipster-registry:8761/config
+            - JHIPSTER_SLEEP=30 # gives time for other services to boot before the application
+            - SPRING_ARTEMIS_HOST=activemq-broker
+        ports:
+            - 8080:8080
+        networks:
+            - artemis
+        depends_on:
+            - artemis-mysql
+            - jhipster-registry
+    usermanagement-app:
+        image: ghcr.io/ls1intum/artemis-usermanagement-service
+        build:
+            context: .
+            dockerfile: ./user-management/src/main/docker/Dockerfile
+        volumes:
+            - ./user-management/src/main/resources/config/application-dev.yml:/home/jhipster/user-management/application-dev.yml:ro
+            - ./src/main/java/resources/config/application-artemis.yml:/home/jhipster/user-management/application-artemis.yml:ro
+        environment:
+            - _JAVA_OPTIONS=-Xmx512m -Xms256m
+            - SPRING_PROFILES_ACTIVE=dev,artemis
+            - MANAGEMENT_METRICS_EXPORT_PROMETHEUS_ENABLED=true
+            - SPRING_DATASOURCE_URL=jdbc:mysql://artemis-mysql:3306/Artemis?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&useUnicode=true&characterEncoding=utf8&useSSL=false&useLegacyDatetimeCode=false&serverTimezone=UTC
+            - EUREKA_CLIENT_SERVICE_URL_DEFAULTZONE=http://admin:$${jhipster.registry.password}@jhipster-registry:8761/eureka
+            - SPRING_CLOUD_CONFIG_URI=http://admin:$${jhipster.registry.password}@jhipster-registry:8761/config
+            - JHIPSTER_SLEEP=30 # gives time for other services to boot before the application
+            - SPRING_ARTEMIS_HOST=activemq-broker
+        ports:
```

Comment:
Are these ports needed outside of the docker network? In other words, should the port 8087 be accessible from the outside world? If not, please use `expose` instead. (Same for all the other `ports` statements). See [here](https://docs.docker.com/compose/compose-file/compose-file-v3/#expose).

Reply:
The ports that should be exposed are the ones of the Gateway, of the Registry (for administration purposes) and the port of the ActiveMQ Artemis console. I've changed that in the file.